static/docs/commands-reference/remote_default.md

```diff
@@ -90,6 +90,10 @@ Change default remote value:
 $ dvc remote default mynewremote
 ```
 
+> Avoid using the same remote that you are using for `dvc push`, `dvc pull`,
```

Comment:
This is not the right place for this note. It should to the external outputs doc. Also, it should be a little bit more informative.

Reply:
need to remove this note from here? and just kept it in the external output section only? @shcheklein

Reply:
yes, I think it's enough to keep that information and notes in the external outputs section